pipeline/transformer/transformer_tei/src/test/scala/weco/pipeline/transformer/tei/TeiTransformerTest.scala

```diff
@@ -41,13 +46,29 @@ class TeiTransformerTest
       Work.Visible[Source](
         version = 1,
         data = WorkData[Unidentified](
+          title = Some("Wellcome Library"),
           description =
-            Some("1 copy of al-Qānūn fī al-ṭibb by Avicenna, 980-1037")
+            Some("1 copy of al-Qānūn fī al-ṭibb by Avicenna, 980-1037"),
+          format = Some(Format.ArchivesAndManuscripts)
         ),
         state = Source(sourceIdentifier, timeModified)
       )
     )
   }
+
+  it("extracts languages") {
+    val teiXml =
+      IOUtils.resourceToString("/Javanese_4.xml", StandardCharsets.UTF_8)
+    val location = createS3ObjectLocation
+    val store =
+      new MemoryStore[S3ObjectLocation, String](Map(location -> teiXml))
+    val transformer = new TeiTransformer(store)
+    val timeModified = instantInLast30Days
+    val id = "Wellcome_Javanese_4"
+    transformer(id, TeiChangedMetadata(location, timeModified), 1).right.get.data.title shouldBe Some(
```

Comment:
Should this be a language-based assertion?

Reply:
err right I think the test name is wrong, will fix